FanGraphSpider.py

```diff
@@ -1,174 +0,0 @@
-                    yield SplashRequest(url, callback=self.steal_parse, args=dict(self.SPLASH_ARGS))
-
-    # Gathers player stats and constructs tweets for homeruns
-    def home_parse(self, response):
-        p_name = response.xpath('//div[@class="player-info-box-name"]/h1/text()').extract_first()
-        team = response.xpath('//div[@class="player-info-box-name-team"]/a/text()').extract_first()
-        av = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[17]/text()').extract_first()
-        rstat = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[7]/text()').extract_first()
-        hr = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[6]/text()').extract_first()
-        rbi = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[8]/text()').extract_first()
-        sb = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[9]/text()').extract_first()
-        obp = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[18]/text()').extract_first()
-        slg = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[19]/text()').extract_first()
-        OPS = str(float(obp)+float(slg))
-
-        tweet = u'\U0001F929' + ' Home Run! ' + u'\U0001F929' + '- %s - %s \n Season Totals \n %s AVG, %s R, %s HR, %s RBI, %s SB, %s OBP, %s OPS \n #homerun #fantasybaseball' % (p_name, team, av, rstat, hr, rbi, sb, obp, OPS)
-
-
-        auth = tweepy.OAuthHandler(tkey, tsecret)
-        auth.set_access_token(atoke, asecret)
-
-        api = tweepy.API(auth)
-
-        api.update_status(tweet)
-
-
-    # Gathers player stats and constructs tweets for Stolen Bases
-    def steal_parse(self, response):
-
-        p_name = response.xpath('//div[@class="player-info-box-name"]/h1/text()').extract_first()
-        team = response.xpath('//div[@class="player-info-box-name-team"]/a/text()').extract_first()
-        av = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[17]/text()').extract_first()
-        rstat = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[7]/text()').extract_first()
-        hr = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[6]/text()').extract_first()
-        rbi = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[8]/text()').extract_first()
-        sb = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[9]/text()').extract_first()
-        obp = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[18]/text()').extract_first()
-        slg = response.xpath('//table[@class="rgMasterTable"][1]/tbody/tr[2]/td[19]/text()').extract_first()
-        OPS = str(float(obp)+float(slg))
-
-        # Composes and Posts Tweet
-        tweet = u'\U0001F3C3' + ' Stolen Base! ' + u'\U0001F3C3' + '- %s - %s \n Season Totals \n %s AVG, %s R, %s HR, %s RBI, %s SB, %s OBP, %s OPS \n #bags #fantasybaseball' % (p_name, team, av, rstat, hr, rbi, sb, obp, OPS)

```

Comment:
I think this tweet line is reused twice and almost exactly the same except for the `'Stolen Base!'` message, so you could pull that out into another function, and to make it easier to read you could use a heredoc: https://lofic.github.io/tips/python-heredoc.html